aten/src/ATen/functorch/BatchRulesReduceOps.cpp

```diff
@@ -401,9 +396,8 @@ TORCH_LIBRARY_IMPL(aten, FuncTorchBatched, m) {
   REDUCTION_BOXED(mode);
   m.impl("nanmedian", nanmedian_decomp);
   REDUCTION_BOXED(nanmedian.dim);
-  // TODO: re-enable these
-  // m.impl("nansum", nansum_decomp);
-  // REDUCTION_BOXED(nansum.dim_IntList);
+  REDUCTION_BOXED(nansum);
+  REDUCTION_BOXED(nanmean);
```

Comment:
nanmean should go into BatchRulesDecomposition -- that's where we put batching rules for CompositeImplicitAutograd functions.

Reply:
Hi, Thanks, @zou3519! I've updated the PR.